devicons/devicon
# Added working workflows that has commenting on forked repo's PR.

This PR is to address this [discussion](https://github.com/devicons/devicon/discussions/383) and issue #478.

To develop this, I cloned the repo and opened a PR from this repo into my fork. This is to ensure that the code properly works with PR from forked repo.

Here's the working [PR](https://github.com/Thomas-Boi/devicon/pull/36). The working results are near the end. Things take up quite a bit of time because of small Actions error like syntaxes and stuff. However, I'm pretty satisfied with the result.

To test:
- Fork this repo including this branch
- Merge this branch into master and develop of the forked repo
- Open a PR from this repo into the forked branch. Make sure that it has the correct criteria of an icon PR (PR title, svgs in the folder, entry in the `devicon.json`. 
  - You can cherry-pick one of the following commits from the branch `testing_files` that I put in this repo. It should be `8ce746` for the valid one (you might need to edit the `devicon.json` file and remove `testingicon1` and `testingicon2`) and `948acf3` for the invalid SVG.
- Tag it 'bot:peek' and watch the show.
